pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -232,17 +239,31 @@ let
       # 4) Patch tensorflow sources to work with later versions of protobuf. See
       #    https://github.com/google/jax/issues/9534. Note that this should be
       #    removed on the next release after 0.3.0.
+      # 5) Patch in https://github.com/bazelbuild/rules_cc/pull/136 to fix the bazel compiler detection on aarch64-darwin
```

Comment:
Thanks for filling this upstream! Since that's done, let's just pull it via `fetchpatch` here instead of copying the patch file into nixpkgs's tree.

Alternatively we can remove it for now since AFAIU the aarch64-darwin build is not yet functional anyhow?

Reply:
I don't think `fetchpatch` will work: the commit is on a PR, so when the PR is merged / closed and the branch deleted it will eventually be garbage collected. I'll remove it then.

Reply:
This is a common pattern in other packages in nixpkgs. It's generally preferred to fetchpatch commits rather than PRs/branches, since PRs are mutable. That's an interesting question re commits being deleted... I'm not sure how that's handled. Perhaps they just link to privately controlled forks of repos? Idk...